docs/_data/portland-2022-config.yaml

```diff
@@ -0,0 +1,227 @@
+    # - text: See the talks!
+    #   link: /speakers
+    # - text: Buy a Ticket!
+    #   link: /tickets
+#    - text: Read our guide
+#      link: /welcome-wagon
+#    - text: See the schedule!
+#      link: /schedule
+    # - text: Watch the Live Stream
+    #   link: /livestream
+    # - text: Browse the photos
+    #   link_absolute: https://www.flickr.com/photos/writethedocs/albums/72157691507514803
+    # - text: Watch the videos!
+    #   link_absolute: https://www.youtube.com/playlist?list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ
+
+tickets:
+  corporate:
+    price: $500
+  independent:
+    price: $275
+  student:
+    price: $100
+
+sponsorship:
+  second_draft:
+    price: $2,000
+  publisher:
+    price: $4,000
+  patron:
+    price: $8,500
+  keystone:
+    price: $15,000
+
+grants:
+  url: https://docs.google.com/forms/d/e/1FAIpQLSdtgQLvwjR7FkVFNMP1qpD3SrUcHqQSPF8PExL-19mxOHrHBQ/viewform
+  ends: "March 1, 2022"
+  notification: "March 10, 2022"
+
+shirts:
+  url: https://teespring.com/wtd-pdx-2022
+  ends: "May 31, 2022"
+
+date: # how do we handle these? human readable would be nice.
+  main: "**May 22-24, 2022**"
+  short: May 22-24, 2022
+  tickets_live: "January 2022"
+  month: May
+
+  # Conference is special for the index to show the combined conference days
+  # conference:
+  #   event: Main Conference
+  #   date: May 23-24
+  #   summary: The main days of the conference. We will be running one main track, and an <a href="/conf/portland/2022/unconference/">Unconference</a>.
+  #   icon: conference
+
+  # These are listed in the schedule
+  day_one:
+    event: Hike
+    date: May 22
+    summary: If you're in town early, join us for the Hike. Portland has excellent access to the woods right in the city, and we'll be doing a short walk through them.
+    icon: hike
+    dotw: Sunday
+  # day_one:
+  #   event: TBD
+  #   date: May 22
+  #   summary: We are still trying to figure out the structure for the first day. It will likely be the hike
+  #   #summary: Join us for the Writing Day and Welcome Reception. The first official day of the conference is full of chances to interact with other documentarians.
+  #   icon: writing
+  #   dotw: Sunday
+  #   writing_day_time: 9am-5pm
+  day_two:
+    event: Conference Day 1
+    date: May 23
+    summary: This is the main event! Hear from lots of interesting folks about all things documentation. We will have talks all day, and unconference sessions running in parallel.
+    icon: conference
+    dotw: Monday
+    talk_time: 9am-5pm
+    unconference_time: 9:45am-5pm
+    social_time: 5pm-7pm
+  day_three:
+    event: Conference Day 2
+    date: May 24
+    summary: And the conference goes on! We are already starting to miss you. |:heart:|
+    icon: unconference
+    dotw: Tuesday
+    talk_time: 9am-4pm
+    job_fair_time: 9:45am-12pm
+    unconference_time: 1:35pm-5pm
+  total_talk_days: 2
+
+about:
+  attendees: around 350
```

Comment:
I see the announcement includes "Hybrid event"...

Eric, I'm not sure if your thinking includes 350 "in person" plus TBD virtual attendees (or 350 total)